src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,53 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: $calendar-size;
+  padding: 10px;
+  margin: 0 auto;
+
+  &__cell {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    @include cells($cell-size);
+    @extend %cell-style;
+    transition-duration: 0.5s;
```

Comment:
Always add transition property. By default, transition property is all, and it's hard for the browser to control all properties